CONTRIBUTING.md

```diff
@@ -3,45 +3,54 @@
+
+ - **Minor Changes** and **Small Features**
+   - Branch: `minor`
+   - Label: _[branch] minor_
+   - Content: Features affecting self-contained components such as modules. Additions to Libraries, API or modules that do not change and function signatures. 
+
+ - **Major Changes**, **Non Backwards-Compatible Changes** and **Large Features**
+   - Branch: `major`
+   - Label: _[branch] major_
+   - Content: Any change modifying a function signature in a library class. Features require extensive LORIS-wide testing. New complex systems and features spanning across multiple modules and libraries. Deprecated functions clean-up.
 
-Your current version number can be found in the VERSION file under the LORIS root.
+For more information about making well-organized pull requests, please read our in-depth Wiki page, ["Contributing to the Code"](https://github.com/aces/Loris/wiki/Contributing-to-the-Code).
 
-## Code Contributions
 
-If you'd like to contribute code, here are some things to keep in mind.
+## Some Things To Keep In Mind
 
 * If your changes require any table modifications, don't forget to modify the
-  SQL/0000*.sql file(s) with your changes for new installs, and also
+  `SQL/0000*.sql` file(s) with your changes for new installs, and also
   include a patch for existing projects to apply to get your changes of which
-  should be placed in the corresponding SQL/VERSIONNUMBER/ directory.
-* Include a test for any new module in the modules/MODULENAME/test/
+  should be placed in the corresponding `SQL/VERSIONNUMBER/` directory.
+* Include a test for any new module in the `modules/MODULENAME/test/`
   directory. You can look at other modules for examples of how to write tests.
   If you have questions, feel free to mail the mailing list.
-* Add your new tests to get auto-run by Travis in the .travis.yml to make sure that
-  other people don't accidentally break your module.
-* Check out our Coding Standards in the [docs/ directory](https://github.com/aces/Loris/tree/master/docs) and also our [Code Review Checklist](https://github.com/aces/Loris/wiki/Code-Review-Checklist) in the [GitHub Wiki](https://github.com/aces/Loris/wiki) and the Pull Request guidelines Readme in this directory. 
-* Try and make sure you run PHP codesniffer using the standards file in
-  docs/LorisCS.xml before sending any pull request, otherwise the Loris tests may
-  fail and we won't be able to merge your pull request.
-* Keep in mind that people are currently using Loris so try and make any changes
-  backwards-compatible with existing installations. If you must change something
+* Add your new automated tests to TravisCI in the `.travis.yml`. 
+* Make sure you run PHP codesniffer using the standards file in
+  `docs/LorisCS.xml` before sending any pull request, otherwise the automated tests will fail.
+* Try and make all changes backwards-compatible with existing installations. If you must change something
```

Comment:
not necessarily non-backwards compatible only. anything that might affect the data and/or code of the project should be flagged (try to formulate that in good english)